tox.ini

```diff
@@ -1,26 +1,18 @@
 [tox]
 envlist =
-    py27-django{18,19,110,111}-{postgresql,sqlite}
-    py35-django{18,19,110,111,20}-{postgresql,sqlite}
+    py35-django{111,20}-{postgresql,sqlite}
```

Comment:
what about adding Django 2.1 as well?

Reply:
I'm on it. :) Actually, I'm adding support to Python3.7 too.